src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/domain/exceptions/TestFeignClientException.java

```diff
@@ -10,7 +10,7 @@ public TestFeignClientException(int status, String message) {
     }
 
     public TestFeignClientException(int status, String message, byte[] responseBody) {
-        super(status, message, responseBody);
+        super(status, message, responseBody, null);
```

Comment:
What is the `null` for here?  Should we be populating with something?

Reply:
I think this should be `Collections.emptyMap()`

Reply:
Passed in responseHeaders